index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'robots-parser';
+
+interface Robot {
+  isAllowed(url: string, ua?: [string]): boolean | undefined;
```

Comment:
Thanks for the PR! Should `[string]` not be `string`?

The UA parameter is optional but should be a string if specified.

Reply:
Oh the `[ua]` in the docs is meant to mean optional, it's based on [JSDoc optional syntax](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values). I might have to see if there's a clearer naming convention for that as it is a bit confusing.

Yeah, the UA should parameter should be a string. Thanks!

Reply:
it appears github updated the pull request